### PR TITLE
Fix request HTTP action with correct method

### PR DESCRIPTION
**Pull request by Brykou**
Use method param in request, and set default value to POST to be compliant with previous uses

### CODE REVIEW

File: lib/HTTP/Action.js [hunk 1]
```diff
@@ -10,12 +10,12 @@ class HTTPAction extends Action {
     this.httpConfig = httpConfig;
   }
 
-  async request(q, method, headers, body) {
+  async request(q, method = 'POST', headers, body) {
```
Comment (mrtbld) [inline lib/HTTP/Action.js]: I think a param with default followed by params without default isn't great. What about putting method, headers and body in a simple object?

Comment (mrtbld) [inline lib/HTTP/Action.js]: Then again body seems to mandatory here, according to how it's used. Should the function allow for bodyless request?

**Comment by mrtbld** [comment]
There's also the problem of the query string being always empty, because inferred from 'path' which never contains query string information.

**Comment by Brykou** [comment]
Needs review

**Comment by antca** [comment]
Looks good. Nothing to add.

File: lib/HTTP/Action.js [hunk 2]
```diff
@@ -14,15 +14,20 @@ class HTTPAction extends Action {
     const path = this.toPath(q);
     const { pathname, query } = url.parse(path, false, true);
     const uri = url.format(Object.assign({}, this.httpConfig, { pathname, query }));
-    return await fetch(uri, {
-      method: 'POST',
+    const request = {
```
Comment (eTorAken) [inline lib/HTTP/Action.js]: does ESLint say anything about the function and this const having the same name?

Comment (Brykou) [inline lib/HTTP/Action.js]: Nothing

**Comment by Brykou** [comment]
Ready for merge ?

**Pull request merged**